localstripe/resources.py

```diff
@@ -751,6 +751,10 @@ def __init__(self, name=None, description=None, email=None,
                 assert set(data.keys()) == {'type', 'value'}
                 assert data['type'] in ('eu_vat', 'nz_gst', 'au_abn')
                 assert type(data['value']) is str and len(data['value']) > 10
+            if payment_method is not None:
+                assert type(payment_method) is str
+                PaymentMethod._api_retrieve(payment_method)
+
```

Comment:
And also this blank line can be removed.

Reply:
oops!

Reply:
Done